main.go

```diff
@@ -31,10 +31,9 @@ func main() {
 
 	// Create application with options
 	err = wails.Run(&options.App{
-		Title:         "SafetyCulture Exporter",
-		Width:         1024,
-		Height:        768,
-		DisableResize: true,
```

Comment:
is everything working ok after this? I mean: if somebody makes the window smaller?

Reply:
Yeah looks okay unless we make the window really small. 


https://user-images.githubusercontent.com/92243746/218865812-5265b8ea-eeb6-4861-abd0-49fb06286bee.mov

Reply:
is Akash and Austin ok with this change?

Reply:
My reason can be the fact a resizing window will loose the feeling of an app. If Austin is ok,.. I have no objections

Reply:
Done.